actions/upload-repository/action.yml

```diff
@@ -45,27 +46,34 @@ runs:
         find build -type f | xargs ls -lh
       shell: bash
 
-    - uses: actions/jekyll-build-pages@44a6e6beabd48582f863aeeb6cb2151cc1716697 # 1.0.13
+    - name: Render repository state in HTML
+      uses: docker://pandoc/core:3.5
       with:
-        source: build
-        destination: build-jekyll
+        args: >-
+          --metadata title="TUF Repository state"
+          --variable title=""
```

Comment:
this weirdness is required so
* the page (tab) has a title and pandoc does not complain
* but the title is not used as a h1 (since we have a header already)